mux/bindings/python/e2e.py

```
@@ -0,0 +1,115 @@
+    deadline = time.time() + timeout
+    old_timeout = stream._conn.sock.gettimeout()
+    stream._conn.sock.settimeout(timeout)
+    try:
+        while time.time() < deadline:
+            stream._conn.sock.settimeout(max(deadline - time.time(), 0.001))
+            event = next(stream)
+            if event.event == "surface-resized" and event.surface == surface:
+                return event
+    finally:
+        stream._conn.sock.settimeout(old_timeout)
+    raise MuxTimeoutError("surface-resized not observed")
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)